pkg/virt-handler/virtwrap/manager.go

```diff
@@ -241,6 +242,11 @@ type LibvirtDomainManager struct {
 	recorder record.EventRecorder
 }
 
+type OvirtDomainManager struct {
+	virConn  Connection
+	recorder record.EventRecorder
```

Comment:
I have been thinking about the record usage in our Manager. It might be better, to move that out of the manager code and do it in the virt-handler controller. Then no k8s interactions are necessary at all in the Manager code. This should be pretty easy when we remove the "defined" event, which is probably a too specific event anyway.